torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -605,7 +617,14 @@ void TensorPipeAgent::pipeRead(
       return;
     }
 
-    auto ctx = createLazyStreamContext();
+    auto ctx = createLazyStreamContext(
```

Comment:
invocation of this occurred 4 times in `tensorpipe_*`. Does it make sense to consolidate this into a helper function and put it in `tensorpipe_utils.*`?

Reply:
addressed